src/index.tsx

```diff
@@ -25,8 +26,14 @@ export class SignatureCanvas extends Component<SignatureCanvasProps> {
     clearOnResize: PropTypes.bool
   }
 
-  static defaultProps: Pick<SignatureCanvasProps, 'clearOnResize'> = {
-    clearOnResize: true
+  static defaultProps: Pick<SignatureCanvasProps, 'clearOnResize' | 'penColor'> = {
+    clearOnResize: true,
+    /**
+     * This ensures that the pen color is set correctly when the parent component re-renders due to state changes and the penColor prop is passed as undefined.
+     * In such cases, the underlying signature_pad assigns fillColor as undefined, preventing drawing.
+     * Ideally, this issue should be fixed in the signature_pad library, but since we are using an older version, we need to handle it here.
+     */
+    penColor: 'black',
```

Comment:
This seems like it could be a breaking change

Reply:
Not a breaking change.
This is inspired from underlying library: https://github.com/szimek/signature_pad/blob/master/src/signature_pad.ts#L96

Reply:
Or rather, the 2.x version: https://github.com/szimek/signature_pad/blob/2171e510b74362a8dd7c8e7fc77a142f16a4aed1/src/signature_pad.js#L24

Although yes, in both cases, upstream does not prevent you from assigning `undefined`. As a wrapper, I think following that behavior and not deviating from it is appropriate.